src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectParameterNames.cs

```diff
@@ -51,6 +53,56 @@ public static class OpenIdConnectParameterNames
         public const string UserId = "user_id";
         public const string Username = "username";
         public const string VersionTelemetry = "x-client-ver";
-#pragma warning restore 1591
     }
+
+    /// <summary>
+    /// Parameter names for OpenIdConnect UTF8 bytes.
+    /// </summary>
+    public static class OpenIdConnectParameterUtf8Bytes
+    {
+        public static readonly byte[] AccessToken = Encoding.UTF8.GetBytes("access_token");
```

Comment:
see this comment: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2208#discussion_r1288896116

Can we use ReadOnlySpan<byte> and u8 strings?

Reply:
nice catch Keegan.

Reply:
@keegan-caruso yes, that is what Stephen wants us to do